specs/binding-kafka.spec/src/main/resources/META-INF/zilla/kafka.idl

```diff
@@ -223,6 +230,7 @@ scope kafka
             KafkaOffset[] progress;     // INIT only
             KafkaKey key;               // INIT only
             KafkaDelta delta;           // INIT + FIN
+            uint64 filters = 0;
```

Comment:
I think this is `INIT only` since it is saying what filter condition(s) matched to send the message, so it would have to be known at the start of the message.

Therefore, I think we can move this up with the other fixed width fields, after timestamp.